frontend/components/App.js

```diff
@@ -33,13 +36,18 @@ class App extends React.Component{
 //     }
   
     //syntax for property
+
     render(){
-        return(
-            <div>
-                 <Navbar/>
+     
+         return (
+         
+         <div>
+             <Navbar />
+                 <ConnectMain/>
```

Comment:
I'm not sure if the main banner should be INSIDE the `Navbar`... I imagine the structure more as such:

```
<Navbar />
<ConnectMain />
```
so it may be better to hoist the `ConnectMain` to a higher level